Files/UserControls/NavigationToolbar/ModernNavigationToolbar.xaml

```diff
@@ -536,7 +536,7 @@
                 FontFamily="Segoe MDL2 Assets"
                 IsEnabled="{x:Bind local1:App.CurrentInstance.NavigationToolbar.CanGoBack}"
                 Style="{StaticResource ToolBarButtonStyle}"
-                ToolTipService.ToolTip="Back">
+                ToolTipService.ToolTip="Back (Alt + Left arrow)">
```

Comment:
We should use a consistent case pattern for `arrow`.

Reply:
you mean this ?: Left Arrow